src/Adapter/Profile/Employee/CommandHandler/BulkDeleteEmployeeHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Profile\Employee\CommandHandler;
+
+use Employee;
+use PrestaShop\PrestaShop\Core\Domain\Profile\Employee\Command\BulkDeleteEmployeeCommand;
+use PrestaShop\PrestaShop\Core\Domain\Profile\Employee\CommandHandler\BulkDeleteEmployeeHandlerInterface;
+
+/**
+ * Class BulkDeleteEmployeeHandler.
+ */
+final class BulkDeleteEmployeeHandler extends AbstractEmployeeHandler implements BulkDeleteEmployeeHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(BulkDeleteEmployeeCommand $command)
+    {
+        foreach ($command->getEmployeeIds() as $employeeId) {
```

Comment:
If one `delete()` fails (throws an exception), then the Command will halt halfway, leaving some of the Employees deleted and other not deleted. A try/catch here would prevent that. But you have to handle the return then 😄 what will you return ? An array of success/errors ? A multi-exception for each failure ?

Reply:
lets throw exception (meaning that we will skip remaining employees) if it fails to delete employee, we have discussed this case with @eternoendless when doing first commands for SQL Manager. :)